metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_config.py

```diff
@@ -119,8 +119,8 @@ class BigQueryV2Config(
     )
 
     match_fully_qualified_names: bool = Field(
-        default=False,
```

Comment:
Won't this break all existing bigquery ingestion where it was not set earlier?

Reply:
I think we've had a deprecation warning since I started... gotta follow up at some point. I'll add a doc change in a follow up though